src/main/kotlin/eu/kevin/api/Endpoint.kt

```diff
@@ -5,11 +5,23 @@ internal object Endpoint {
     const val VERSION = "/v0.3"
 
     object Path {
-        fun initiatePayment() = "/pis/payment"
-        fun getPaymentStatus(paymentId: String) = "/pis/payment/$paymentId/status"
-        fun initiatePaymentRefund(paymentId: String) = "/pis/payment/$paymentId/refunds"
-        fun startAuthentication() = "/auth"
-        fun receiveToken() = "/auth/token"
-        fun receiveTokenContent() = "/auth/token/content"
+        object Payment {
```

Comment:
This name refers scope/component and usually scope is plural. I would name this as `Payments`

Reply:
I would not name it **Endpoint** as well, but something like **Api.Paths...**. etc.